WalletWasabi.Fluent/ViewModels/Wallets/WalletSettingsViewModel.cs

```diff
@@ -84,4 +101,25 @@ public WalletSettingsViewModel(WalletViewModelBase walletViewModelBase)
 	public override sealed string Title { get; protected set; }
 
 	public ICommand VerifyRecoveryWordsCommand { get; }
+
+	private void ValidatePlebStopThreshold(IValidationErrors errors) =>
+		ValidatePlebStopThreshold(errors, PlebStopThreshold);
+
+	private static void ValidatePlebStopThreshold(IValidationErrors errors, string plebStopThreshold)
+	{
+		if (!string.IsNullOrWhiteSpace(plebStopThreshold))
```

Comment:
nit, feel free to ignore it.

IMO better to use 
```
if (string.IsNullOrWhiteSpace(plebStopThreshold))
{
	return;
}
```

It reduces nesting and easier to read the code.